docs/en/reference/sql/dql/LIMIT_CLAUSE.md

```diff
@@ -16,11 +16,12 @@ SELECT ... LIMIT ...
 ```
 
 ## Description
-For the standalone version, `LIMIT` is supported in all conditions. For the cluster version, the execution modes, which support this clause, are shown below.
+
+Limit clause is not supported in online preview mode. By practise, limit clause should used together with order by clause, to constrains the result rows into a unique order. OpenMLDB do not support order by clause yet, it is exepcted to get inconsistent results for the same query.
```

Comment:
not supported in online request mode?

Reply:
since request mode always return one row. It is useless for limit appear in the end of sql string. So it is turned off in case confusing. 

Using limit inside a subquery has its meaning for request mode, but we do not support it yet, physical node is not optimized for limit subquery

Reply:
then here is a typo, right? `online preview mode`